src/bot/scrapper/scrapperBase.ts

```diff
@@ -0,0 +1,38 @@
+import puppeteer, { Browser, Page } from 'puppeteer';
+
+export abstract class ScrapperBase {
```

Comment:
**nitpick:** Abstract class is not for that purpose: https://khalilstemmler.com/blogs/typescript/abstract-class/